octobot_trading/exchanges/implementations/margin_exchange_simulator.py

```diff
@@ -13,12 +13,17 @@
 #
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
+import typing
 import octobot_trading.exchanges.connectors as exchange_connectors
 import octobot_trading.exchanges.types as exchanges_types
+from octobot_trading.exchanges.config import ccxt_exchange_settings
 
 
 #TODO remove
 class MarginExchangeSimulator(exchanges_types.MarginExchange):
+    connector_config: ccxt_exchange_settings.CCXTExchangeConfig = (
+        ccxt_exchange_settings.CCXTExchangeConfig
```

Comment:
I don't understand why we have ccxt stuff in the generic margin exchange implementation